main.go

```diff
@@ -118,10 +114,11 @@ func New(config ...Config) fiber.Handler {
 			return cfg.ErrorHandler(c, err)
 		}
 
+		// first set the ContextKey as Locals so the Validator can be checked correctly
+		c.Locals("ContextKey", cfg.ContextKey)
```

Comment:
what is the reason for this line? we should store if then the real name of the context key inside the locals

the validation function should then also know this name and not refer to something general, which only implies that the workflow has been run

if you use this middleware on different routes this can lead to collisions from my point of view